src/components/UserNav/NotificationCard/Invitation/index.js

```diff
@@ -0,0 +1,66 @@
+import {useCallback} from 'react';
+import {BsPersonPlusFill} from 'react-icons/bs';
+
+import Button from 'components/Button';
+import List from '@ra/components/List';
+import NoNotification from '../NoNotification';
+
+import styles from './styles.scss';
+
+const Invitation = ({invitations}) => {
+    const handleAccept = useCallback(() => {
+        alert('Accepted');
+    }, []);
+
+    const handleDecline = useCallback(() => {
+        alert('Declined');
+    }, []);
+
+    const renderInvitation = useCallback(
+        ({item}) => {
+            return (
+                <div className={styles.invitation} key={item.id}>
+                    <div className={styles.iconContainer}>
+                        <BsPersonPlusFill
+                            size={20}
+                            className={styles.personIcon}
+                        />
+                    </div>
+                    <div className={styles.descriptionContainer}>
+                        <p className={styles.description}>{item.description}</p>
+                        <div className={styles.buttons}>
+                            <Button
+                                className={styles.button}
+                                type='button'
+                                onClick={handleAccept}
+                            >
+                                Accept
+                            </Button>
+                            <Button
+                                className={styles.button}
+                                type='button'
+                                secondary
+                                onClick={handleDecline}
+                            >
+                                Decline
+                            </Button>
+                        </div>
+                    </div>
+                </div>
+            );
+        },
+        [handleAccept, handleDecline]
+    );
+
+    return (
+        <List
+            data={invitations}
+            renderItem={renderInvitation}
+            keyExtractor={(item, index) => index}
+            className={styles.container}
+            emptyComponent={<NoNotification text='No Invitations' />}
```

Comment:
use memoized value. Passing new object to prop will create new instance of it in every render and decreases performance of system.